ext_localconf.php

```diff
@@ -32,6 +32,9 @@
     \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig('<INCLUDE_TYPOSCRIPT: source="FILE:EXT:bootstrap_package/Configuration/TsConfig/Page/ContentElement/All.tsconfig">');
 }
 
+// BackendLayouts to the Web Info Module
+\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTSConfig('<INCLUDE_TYPOSCRIPT: source="FILE:EXT:bootstrap_package/Configuration/TsConfig/Page/Mod/WebInfo/WebInfo.tsconfig">');
```

Comment:
@benjaminkott I don't think we need the possibility to disable this like for the other PageTSConfig options

Reply:
I agree that we do not need this kind of option, but should we not add this to the core?

Reply:
I'm not sure. For BP backend layouts are an important part and that must not be the case for every TYPO3 installation. @bmack what do you think?

Reply:
feel free to create a FEATURE for core to add this.